effekt/jvm/src/main/scala/effekt/Server.scala

```diff
@@ -180,7 +180,7 @@ class Server(config: EffektConfig, compileOnChange: Boolean=false) extends Langu
    */
   def publishHoles(source: Source, config: EffektConfig)(implicit C: Context): Unit = {
     if (!workspaceService.settingBool("showHoles")) return
-    val holes = getHoles(source)
+    val holes = getHoles(source).toArray
```

Comment:
Is this still needed, since you hooked into gson? I would prefer not to use `scala.Array` if we can help it.